### PR TITLE
Create merge job for coverage

### PR DESCRIPTION
**Pull request by MarionJHolloway**
# Description
Create a job to merge the reconciliation dataset with cqc locations data to create data for coverage

# How to test
Unit tests passing
Run in branch: https://eu-west-2.console.aws.amazon.com/states/home?region=eu-west-2#/v2/executions/details/arn:aws:states:eu-west-2:344210435447:execution:create-merge-job-for-coverage-Ind-CQC-Filled-Post-Estimates-Pipeline:357d3d02-141c-4598-ada7-336502ed27b7

# Developer checklist
- [ ] Unit test
- [ ] Linked to Trello ticket
- [ ] Documentation up to date

### CODE REVIEW

File: jobs/merge_coverage_data.py [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+    CQCLClean.number_of_beds,
+    CQCLClean.regulated_activities,
+    CQCLClean.gac_service_types,
+    CQCLClean.services_offered,
+    CQCLClean.specialisms,
+    CQCLClean.primary_service_type,
+    ONSClean.contemporary_ons_import_date,
+    ONSClean.contemporary_cssr,
+    ONSClean.contemporary_icb,
+    ONSClean.contemporary_region,
```
Comment (garypolzin) [inline jobs/merge_coverage_data.py:42]: we could scrap these columns, will only ever need coverage for geogs that exist now - hardly game changing but 4 columns less at least!

File: jobs/merge_coverage_data.py [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+    Keys.day,
+    Keys.import_date,
+]
+cleaned_ascwds_workplace_columns_to_import = [
+    AWPClean.ascwds_workplace_import_date,
+    AWPClean.location_id,
+    AWPClean.establishment_id,
+    AWPClean.organisation_id,
+    AWPClean.total_staff_bounded,
+    AWPClean.worker_records_bounded,
```
Comment (garypolzin) [inline jobs/merge_coverage_data.py:59]: might be better to import the original versions of these

**Review by garypolzin — COMMENTED** [review]
I know it's still WIP but added a few minor comments, otherwise it looks great!

File: jobs/merge_coverage_data.py (hunk 1 above)
Comment (MarionJHolloway) [reply jobs/merge_coverage_data.py]: I've removed these.

File: jobs/merge_coverage_data.py (hunk 2 above)
Comment (MarionJHolloway) [reply jobs/merge_coverage_data.py]: Yup, this was breaking the job because they don't exist in the reconciliation dataset.

**Review by garypolzin — APPROVED** [review]
looks great thanks!

**Pull request merged**